src/indra_cogex/client/enrichment/discrete.py

```diff
@@ -96,6 +96,7 @@ def count_human_genes(*, client: Neo4jClient) -> int:
     query = """\
         MATCH (n:BioEntity)
         WHERE n.id STARTS WITH 'hgnc'
+        AND NOT n.obsolete = "True"
```

Comment:
we should check with @kkaris if we've updated this import entry to be a proper boolean yet or if it's still considered as a string. Then it could be updated to simply

```suggestion
        AND NOT n.obsolete
```

Reply:
`AND NOT n.obsolete` wouldn't work just like that since we have "False" and "True" strings for every node.